config/google_cloud_exporter/postgresql/README.md

```diff
@@ -0,0 +1,41 @@
+# Postgresql Metrics with Google Cloud
+
+The Postgresql Receiver can be used to send Postgresql metrics to Google Cloud Monitoring.
+
+## Limitations
+
+The collector must be installed on the Postgresql system.
+
+## Prerequisites
+
+See the [documentation](https://github.com/observIQ/observiq-otel-collector/blob/main/docs/receivers.md) for Postgresql prerequisites.
```

Comment:
Not sure if we want to link to the PG-specific README like below (https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/postgresqlreceiver/README.md)